sources/packages/backend/package.json

```diff
@@ -124,6 +127,7 @@
     "coveragePathIgnorePatterns": [
       "node_modules",
       "db-migrations",
+      "workflow",
```

Comment:
Thanks for adding it. Should we be more precise, maybe using `backend/workflow`?